src/magmaviz/boxplot.py

```diff
@@ -20,5 +24,52 @@ def boxplot(df, x, y, facet=False):
     Examples
     --------
     >>> from magmaviz.magmaviz import boxplot
-    >>> boxplot(mtcars, horsepower, cars, facet=True)
+    >>> boxplot(cars, "Miles_per_Gallon", "Origin", facet=True)
     """
+
```

Comment:
Just a thought. Do you want to make it more flexible by adding another column to sub-categorize the box plot by? Like for e.g. boxplot of "Miles_per_Gallon" on x, "Origin" on y, and further differentiating/categorizing the boxplots by 'continent' as color?